### DISCUSSION TITLE
code-server for teams?

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5828] [Q&A]
I'm writing a blog post about different setups for using code-server with multiple users and would love to showcase some from the community.

If you have a solution and it's open source, can you share it in the comments below? 🙏🏼

Thank you!

**Comment by jsjoeio** [comment]
Here's the link in case anyone wants to review it: https://docs.google.com/document/d/1JD0c4r9f0Zv4P8VGWBMCHH8PrgtknHEAFZLaHWRwTIM/edit?usp=sharing

**Comment by tobycm** [comment]
https://github.com/tobycm/code-server-instance-manager-nginx

**Reply by tobycm** [reply]
Part 2: https://github.com/tobycm/nginx-code-server-router/

**Reply by jsjoeio** [reply]
Huzzah! Thank you :D

**Comment by jsjoeio** [comment]
It's live! https://coder.com/blog/code-server-multiple-users
